package.json

```diff
@@ -233,7 +233,7 @@
     "moment": "^2.29.4",
     "moment-timezone": "^0.5.46",
     "mysql2": "^3.11.4",
-    "next": "^16.0.1",
+    "next": "^16.0.7",
```

Comment:
I think you need to merge master into this PR (and also rerun the codegen, based on the merge conflict)